tools/build-prod-env.js

```diff
@@ -15,6 +15,8 @@ if (!_.isEqual(_.intersection(_.keys(argv), requiredParams), requiredParams)) {
 
 fs.copyFileSync(path.resolve(__dirname, '../.sample.env'), process.env.NODE_ENV === 'production' ? path.resolve('/etc', 'lamassu', '.env') : path.resolve(__dirname, '../.env'))
 
+setEnvVariable('NODE_ENV', 'production')
```

Comment:
This is problematic. We're not really updating this file with lamassu-update, also we can't just rewrite everything, we need a "migration" mechanism for this. This is also a risky thing to migrate since the database password gets stored here, so maybe leaving backups in case of migration is a good idea.

Reply:
Added the mechanism on a new commit